tests/test_metadata_creation.py

```diff
@@ -218,5 +221,48 @@ def test_no_response_warning(
         self.assertIsNone(actual_subject)
 
 
+class TestDataDescriptionMetadata(unittest.TestCase):
+    """Tests methods in DataDescriptionMetadata class"""
+
+    conf_file_path = CONFIGS_DIR / "ephys_upload_job_test_configs.yml"
+    args = ["-c", str(conf_file_path)]
+    loaded_configs = EphysJobConfigurationLoader().load_configs(args)
```

Comment:
It doesn't look like you're using any of these variables in this class? You can probably safely remove them.